app/move/fields/create.js

```diff
@@ -381,6 +381,34 @@ module.exports = {
       text: 'fields::documents.hint',
     },
   },
+  prison_transfer_reason: {
+    validate: 'required',
+    component: 'govukRadios',
+    name: 'reason',
```

Comment:
This needs to match the key. It's one of the slightly annoying things to keep an eye out for, but its because we use this object to pass to the GOV.UK Design System components.